src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure.mdx

```diff
@@ -14,14 +14,20 @@ redirects:
 
 import infrastructureInventoryNameSourcePath from 'images/infrastructure_screenshot-full_inventory-name-source-path.webp'
 
-New Relic can collect detailed information about a system's configuration per host, including system modules, configuration files, metadata, packages, services, user sessions, etc. The **Inventory** page provides a real-time, filterable, searchable view into each host's configuration.
+New Relic can collect detailed information about a system's configuration per host, including system modules, configuration files, metadata, packages, services, user sessions, and more. The **Inventory** page provides a real-time, filterable, searchable view into each host's configuration.
 
 Use the **Inventory** page to:
 
 * Ensure a version update was applied successfully across all your hosts.
 * Audit version discrepancies across your hosts.
 * Quickly identify which hosts require an update to fix a security vulnerability.
 
+## What data is reported? [#data]
```

Comment:
@zuluecho9 this section doesn't really explain which data is reported. Honestly, this current page seems to explain what users need to know about which data is reported. Maybe we need to change this section to make it a reminder to install the infra agent to ensure they get their inventory?